_includes/footer.html

```diff
@@ -1,9 +1,7 @@
 <div class="footer row">
     <div class="footer-col col-md-6">
-        <a href="{{site.url}}" class="img-link">
-            <img id="footer-logo" class="center img-fluid" src="{{site.url}}/img/logo-monochrome-dark.png" alt="FAMNM"/>
-        </a>
-        <div id="copyright-text">&copy; 2019 <span class="italic">FIRST</span> Alumni and Mentors Network at Michigan</div>
+        <img id="footer-logo" class="center img-fluid h-auto" src="{{site.url}}/img/logo-monochrome-dark.png" alt="FAMNM"/>
```

Comment:
Is there a reason we decide to remove the link from this?

Reply:
we decided to remove the link from this because it didn't make sense from a UX perspective - the navbar is fixed to the top of the user's viewport so even on mobile the top left corner serves as a link to the homepage.